rocketpy/simulation/flight.py

```diff
@@ -1768,34 +1768,42 @@ def u_dot_parachute(self, t, u, post_processing=False):
             e0dot, e1dot, e2dot, e3dot, alpha1, alpha2, alpha3].
 
         """
-        # Parachute data
-        cd_s = self.parachute_cd_s
-        ka = 1
-        R = 1.5
-        rho = self.env.density.get_value_opt(u[2])
-        to = 1.2
-        ma = ka * rho * (4 / 3) * np.pi * R**3
-        mp = self.rocket.dry_mass
-        eta = 1
-        Rdot = (6 * R * (1 - eta) / (1.2**6)) * (
-            (1 - eta) * t**5 + eta * (to**3) * (t**2)
-        )
-        Rdot = 0
         # Get relevant state data
-        x, y, z, vx, vy, vz, e0, e1, e2, e3, omega1, omega2, omega3 = u
-        # Get wind data
+        z, vx, vy, vz = u[2:6]
+
+        # Get atmospheric data
+        rho = self.env.density.get_value_opt(z)
         wind_velocity_x = self.env.wind_velocity_x.get_value_opt(z)
         wind_velocity_y = self.env.wind_velocity_y.get_value_opt(z)
-        free_stream_speed = (
-            (wind_velocity_x - vx) ** 2 + (wind_velocity_y - vy) ** 2 + (vz) ** 2
-        ) ** 0.5
+
+        # Get Parachute data
+        cd_s = self.parachute_cd_s
+
+        # Get the mass of the rocket
+        mp = self.rocket.dry_mass
+
+        # Define constants
+        ka = 1  # Added mass coefficient (depends on parachute's porosity)
+        R = 1.5  # Parachute radius
+        # to = 1.2
+        # eta = 1
+        # Rdot = (6 * R * (1 - eta) / (1.2**6)) * (
+        #     (1 - eta) * t**5 + eta * (to**3) * (t**2)
+        # )
+        # Rdot = 0
+
+        # Calculate added mass
+        ma = ka * rho * (4 / 3) * 3.141592653589793 * R**3
```

Comment:
Use `np.pi` or `math.pi`

Reply:
It was using np.pi before, I thought it would be faster to use the value directly. Any objections?

Reply:
ok, done: [f07be5e](https://github.com/RocketPy-Team/RocketPy/pull/596/commits/f07be5ee023ec1750c6017f3b97ab7b81f161169)